server/src/routes.py

```diff
@@ -20,13 +22,16 @@ def setup_routes(app):
                 json_msg = aiohttp.WSMessage.json(msg)
                 if json_msg['fname'] == 'closeSocket':
                     print('Closing socket')
+                    message = MessageSingleton("main socket closed")
                     await ws.close()
                 elif json_msg['fname'] == 'getNetworkInfo':
                     print('Getting network info')
+                    message = MessageSingleton("getting network info")
                     json_iface = json.dumps(get_interface())
                     await ws.send_str(json_iface)
                 elif json_msg['fname'] == 'arpScan':
                     print('Getting arp table')
+                    message = MessageSingleton("getting arp table")
```

Comment:
same with this. i wonder if it has to do with it being a different endpoint?

Reply:
I think I put this and the network info one in, found they didn't work, and then forgot to take them out. I replaced the ARP table call deeper but I am confused now why these aren't working, why are they getting skipped over?